src/VisualStudio/Core/Def/Implementation/Utilities/VisualStudioWaitIndicator.cs

```diff
@@ -48,17 +49,9 @@ public VisualStudioWaitIndicator(SVsServiceProvider serviceProvider)
                 {
                     return WaitIndicatorResult.Canceled;
                 }
-                catch (AggregateException e)
+                catch (AggregateException aggregate) when (aggregate.InnerExceptions.All(e => e is OperationCanceledException))
```

Comment:
The fact this is catching AggregateException at all seems really fishy. Did you get any sense while you were here that this was actually being taken...?